scripts/coverage_report.sh

```diff
@@ -16,9 +16,10 @@ TEST_DIR=$1
 COVERAGE_DIR=$2
 
 # This needs to run in libra
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-if [ ! $(echo $DIR | grep -Ee ".*/scripts") ]; then
-	echo "This needs to run in libra, not in $(pwd)"
+SCRIPT_DIR="$( dirname "${BASH_SOURCE[0]}" )"
```

Comment:
Believe me, I wish this was not arcane.  But it is. :(

```
fga@fga-ubuntu-PC0UYPZ9➜~» ls -l toto.sh                                                                                                                                                    [15:28:02]
-rwxr-xr-x 1 fga fga 132 Jul 26 15:23 toto.sh
fga@fga-ubuntu-PC0UYPZ9➜~» ls -l bin/toto.sh                                                                                                                                                [15:28:06]
lrwxrwxrwx 1 fga fga 10 Jul 26 15:24 bin/toto.sh -> ../toto.sh
fga@fga-ubuntu-PC0UYPZ9➜~» cat toto.sh                                                                                                                                                      [15:28:11]
#!/bin/bash
DIR1="$( dirname "${BASH_SOURCE[0]}" )"
DIR2="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"

echo $DIR1
echo $DIR2
fga@fga-ubuntu-PC0UYPZ9➜~» ~/toto.sh                                                                                                                                                        [15:28:16]
/home/fga
/home/fga
fga@fga-ubuntu-PC0UYPZ9➜~» ~/bin/toto.sh                                                                                                                                                    [15:28:31]
/home/fga/bin
/home/fga/bin
fga@fga-ubuntu-PC0UYPZ9➜~» cd ..                                                                                                                                                            [15:28:37]
fga@fga-ubuntu-PC0UYPZ9➜/home» ~/toto.sh                                                                                                                                                    [15:28:39]
/home/fga
/home/fga
fga@fga-ubuntu-PC0UYPZ9➜/home» ~/bin/toto.sh                                                                                                                                                [15:28:42]
/home/fga/bin
/home/fga/bin
fga@fga-ubuntu-PC0UYPZ9➜/home» ./fga/toto.sh                                                                                                                                                [15:29:07]
./fga
/home/fga

```

=> you don't get what you want if the script is not called with a relative path. OTOH, computation *from* DIR2 as defined above might give you what you want.